src/App.js

```diff
@@ -5,27 +5,27 @@ import Header from "./components/Header/Header";
 import Add from "./Pages/Add";
 import Vote from "./Pages/Vote";
 import Result from "./Pages/Result";
+
+// import BottomImg from "./temple.jpg";
 // import "./components/Header";
```

Comment:
I shall comment this comment: pls no comments in the code unless you are taking notes in a WIP or writing a tutorial or exercise
:>